theme/static/style/style.css

```diff
@@ -98,6 +98,113 @@ ul.topnav li [name="q"] {
     border: 0;
 }
 
+.topnav{
+    padding: 10px;
+    overflow: hidden;
+    box-shadow: 0 2px 5px 0 rgba(0,0,0, .3);
+    z-index: 1;
+}
+
+.topnav ul{
+    list-style-type: none;
+    text-align: right;
+}
+
+.topnav li{
+    display: inline-block;
+    margin-left: 15px;
+    vertical-align: top;
+}
+
+.topnav ul a{
+    display: inline-block;
+    padding: 0 16px;
+    text-align: center;
+    line-height: 50px;
+    border-radius: 3px;
+}
+
+.topnav .searchbox{
```

Comment:
The block should refer to `.topnav .search`, in order for its styles to apply. Same goes for the block at line `92`. If You're going to use this one instead of the block at line `92`, then you'll need to replace the `px` units with ``rem`` or any other units that are commonly used in this stylesheet.